src/actions/pubSubHandlers.js

```diff
@@ -248,26 +248,22 @@ export function handleRemoveRoom({ channel: id }) {
   }
 }
 
-const changeUserStatue = payload => dispatch => {
+const changeUserStatus = payload => dispatch => {
   dispatch({
     type: types.CHANGE_USER_STATUS,
     payload,
   })
 }
 
-export const handleUserStatusChange = ({ status, user: userId }) => (
+export const handleUserStatusChange = ({ status, user: id }) => (
   dispatch,
   getState,
 ) => {
   const users = usersSelector(getState())
-  const user = find(users, { id: userId })
+  const user = find(users, { partner: { id } })
   if (user) {
-    dispatch(changeUserStatue({ status, userId }))
-    return
+    dispatch(changeUserStatus({ status, id }))
   }
-  dispatch(addNewUser(userId)).then(() => {
```

Comment:
Why could you remove this case? Has `addNewUser` always been called once `handleUserStatusChange` is invoked?

Reply:
@nikgraf unfortunately almost yes:( Because in `users` array we now store not all users, and if it will be a new user, there is no reason to add him to `users` because anyway, we will not see status changing for users with which we didn't have conversation